<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindow</class>
 <widget class="QDialog" name="chatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/background/res/chatWindow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>458</y>
     <width>71</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
border-top-left-radius:5px;
border-bottom-left-radius:5px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>发送信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>393</y>
     <width>556</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 20%);</string>
   </property>
   <property name="placeholderText">
    <string>请输入待发送消息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>556</width>
     <height>338</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
background-color: rgba(255, 255, 255, 20%);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>8</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ipLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>PORT:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>inputEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>sendButton</zorder>
  <zorder>showEdit</zorder>
  <zorder>usernameLabel</zorder>
  <zorder>ipLabel</zorder>
  <zorder>portLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
